data/json/encounters/randec_independent_travelers.json

```
@@ -0,0 +1,102 @@
+[
+  {
+    "type": "effect_on_condition",
+    "id": "EOC_RandEnc_Roadstop_camper_add",
+    "recurrence": 1800,
+    "global": true,
+    "effect": [
+      { "set_condition": "random_enc_condition", "condition": { "math": [ "0", "==", "0" ] } },
+      {
+        "run_eoc_with": "EOC_RandEnc",
+        "variables": {
+          "omt": "roadstop_a",
+          "map_update": "nest_RandEnc_campervan_traveler_add",
+          "map_removal": "nest_RandEnc_campervan_traveler_remove",
+          "chance": "10",
+          "days_till_spawn": "7"
+        }
+      }
+    ]
+  },
+  {
+    "type": "effect_on_condition",
+    "id": "EOC_traveler_random_destination_direction",
+    "//": "Picks a random direction for the NPC in question to be going in.  Used for dialogue checks.",
+    "condition": { "not": { "u_has_var": "picked_direction", "type": "dialogue", "context": "travel_direction", "value": "yes" } },
+    "effect": [
+      {
+        "//": "Most people aren't going east because of the big cities with lots of zombies, so that response isn't as likely to come up.",
+        "weighted_list_eocs": [
+          [
+            {
+              "id": "going_north",
+              "effect": [
+                { "u_add_var": "going_north", "type": "dialogue", "context": "travel_direction", "value": "yes" },
+                { "u_add_var": "picked_direction", "type": "dialogue", "context": "travel_direction", "value": "yes" }
+              ]
+            },
+            { "const": 1 }
```

weighted_list_eocs allows int
```suggestion
            1
```